### ISSUE TITLE
Allow multiple colorpicks in a row

### ISSUE THREAD

**Issue by ProLoser** [kepi/chromeEyeDropper#26]
Since you're making use of a history feature, it would be nifty if you could select multiple colors on the page at a time, and then go review them. What if you could hold down (or toggle with) [SHIFT] while picking colors so it doesn't immediately deactivate. You can select multiple colors and then hit shift again before choosing your last color or just exit colorpicker mode.

**Comment by jenmarketing** [comment]
Agree this could be a great addition if you are still updating this tool

**Comment by kepi** [comment]
Yes, I'm even. Even when it doesn't look like it from this particular issue :) I still want to add this feature.
